test/unit/test_email.py

```diff
@@ -30,3 +31,12 @@ def test_send_without_login(smtp):
                   "group": "group1",
                   "touchstone": "tid"})
     smtp.return_value.login.assert_not_called()
+
+
+@patch("src.utils.email.logging")
+def test_send_handles_exceptions(logging):
+    send_email(None, None, None, None, None, None)
+    expected_error = AttributeError(
+        "'NoneType' object has no attribute 'success_email_recipients'")
```

Comment:
Is there any context on the log? Or is it just the message? Might just be my familiarity with the code base but I think I would find it hard to work out where the error was coming from from this message alone.

Reply:
Valid question but python's `logging.exception` method automatically includes the stack trace, so shouldn't be a problem :+1: